objects/ui2/ui2Event.self

```diff
@@ -550,7 +550,7 @@ Feel free to inherit me and override the ones you can implement.
             | 
             combo nonmodifierKeyCap isArrow ifTrue: [^ handleSmallArrowPress: combo IfCannot: b].
 
-            combo nonmodifierKeyCap isPrintable ifTrue: [^combo nonmodifierKeyCap printString do: [|:c| insert_char: c]].
+            combo nonmodifierKeyCap printString size <= 6 ifTrue: [^combo nonmodifierKeyCap printString do: [|:c| insert_char: c]].
```

Comment:
Longest UTF-8 byte sequence.

In case of unknown event, there is name of the event as string - 'keyCaps unknown' for example. <= 6 never happens, because the string 'keyCaps' is always longer.

I don't say it is nice or anything, but it can detect unicode input in simplest possible way. Of course, if anyone knows better way, I am all for it.